test/steps/kogitoapp.go

```diff
@@ -30,7 +30,7 @@ func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 	s.Step(`^Deploy (quarkus|springboot) example service "([^"]*)" with configuration:$`, data.deployExampleServiceWithConfiguration)
 	s.Step(`^Create (quarkus|springboot) service "([^"]*)"$`, data.createService)
 	s.Step(`^Create (quarkus|springboot) service "([^"]*)" with configuration:$`, data.createServiceWithConfiguration)
-	s.Step(`^Deploy service from example file "([^"]*)"$`, data.deployServiceFromExampleFile)
+	s.Step(`^Deploy (quarkus|springboot) service from example file "([^"]*)"$`, data.deployServiceFromExampleFile)
```

Comment:
Could you please move this change into separate PR?
It doesn't seem to be related to the JIRA.

Reply:
I had to make this change in here since we need to set the runtimeType now in the constructor of the KogitoAppStub, so we can resolve the images to use accordingly based on the build type and the runtime type.

Reply:
Oh, ok, I have overlooked that.

Reply:
Also, this step is not used anywhere, but I thought it was better to maintain it just in case you're creating your own scenarios for testing purposes.